extension/popup.js

```diff
@@ -20,11 +20,11 @@
 
 function saveChanges() {
 	var keywords = document.getElementById("keywords").value;
-	var interrupt = document.getElementById('chk-interrupt').checked;
+    var interrupt = document.getElementById('chk-interrupt').checked;
 
 	localStorage["uget-keywords"] = keywords;
 
-	chrome.runtime.getBackgroundPage(function(backgroundPage) {
+	browser.runtime.getBackgroundPage(function(backgroundPage) {
```

Comment:
Same issue as in `background.js`. Need to change `browser` by a variable that changes based on the browser type.